__tests__/__snapshots__/App-test.js.snap

```diff
@@ -32394,6 +32394,7 @@ exports[`ProgressView Example Page 1`] = `
               }
             >
               <RNCProgressViewWindows
+                accessibilityLabel="Simple ProgressView"
```

Comment:
FYI the snapshot changes seem unrelated to this PR. Missed from an earlier change?